src/libs/settings/creporterprivacysettingsmodel.cpp

```diff
@@ -195,6 +201,21 @@ bool CReporterPrivacySettingsModel::allowMobileData() const
     return value(Settings::AllowMobileData, QVariant(false)).toBool();
 }
 
+bool CReporterPrivacySettingsModel::restrictWhenLowBattery() const
+{
+    return value(Settings::RestrictWhenLowBattery, QVariant(true)).toBool();
+}
+
+bool CReporterPrivacySettingsModel::restrictWhenDischarging() const
+{
+    return value(Settings::RestrictWhenDischarging, QVariant(true)).toBool();
+}
+
+int CReporterPrivacySettingsModel::dischargingThreshold() const
+{
+    return value(Settings::DischargingThreshold, QVariant(30)).toInt();
```

Comment:
10 is the low-battery state which is relevant for the RestrictWhenLowBattery option. DischargingThreshold is relevant for the RectrictWhenDischarging option, which is second level of power saving. First level exists to prevent boot failure on cable. Second level is meant to prevent unwanted discharging while not "at home" (you take your device where you will not be able to charge it for longer time and you do not want to sacrifice you battery power to crash reporter accidentaly reporting a lot).

Reply:
Core dumps for web browser are over 100MB. In any case this is quite subjective I think. It depends on how/where you use the device - I do care, you don't. If something is subjective, making it configurable adds value.